pages/student-team.tsx

```diff
@@ -10,159 +10,117 @@ import { media } from 'utils/media';
 const TeamMembers = [
   {
     name: 'Gyanendra Kumar',
-    image: '/testimonials/iitk-logo.svg',
-    phone: '(+91) 78965 78965',
-    mail: 'xyz@iitk.ac.in',
-    linkedin: 'https://www.linkedin.com/in/abc'
+    image: '/testimonials/Gyanendra.jpg',
```

Comment:
check `.jpg` everywhere, some images have `.JPG`

Reply:
rename file names not literals